backend/src/services/memberService.ts

```diff
@@ -862,22 +868,41 @@ export default class MemberService extends LoggerBase {
     const relationships = ['tags', 'notes', 'tasks', 'identities', 'affiliations']
 
     try {
-      const member = await MemberRepository.findById(memberId, this.options, {
-        doPopulateRelations: 'no-activity-aggregates',
-      })
-
-      member.memberOrganizations = await MemberOrganizationRepository.findMemberRoles(
-        memberId,
-        this.options,
+      const qx = SequelizeRepository.getQueryExecutor(this.options)
+      const memberById = await findMemberById(qx, memberId, [
+        MemberField.ID,
+        MemberField.TENANT_ID,
+        MemberField.DISPLAY_NAME,
+        MemberField.ATTRIBUTES,
+        MemberField.REACH,
+        MemberField.CONTRIBUTIONS,
+        MemberField.MANUALLY_CHANGED_FIELDS,
+      ])
+
+      const [memberOrganizations, identities, tags, notes, tasks, affiliations] = await Promise.all(
+        [
+          MemberOrganizationRepository.findMemberRoles(memberId, this.options),
+          fetchMemberIdentities(qx, memberId),
+          findMemberTags(qx, memberId),
+          findMemberNotes(qx, memberId),
+          findMemberTasks(qx, memberId),
+          findMemberAffiliations(qx, memberId),
+        ],
       )
 
-      // check member has the sent identity
-      const memberIdentities = (await MemberRepository.getIdentities([memberId], this.options)).get(
-        memberId,
-      )
+      const member = {
+        ...memberById,
+        memberOrganizations,
+        identities,
+        affiliations,
+        tags: tags.map((t) => ({ id: t.tagId })),
+        notes: notes.map((n) => ({ id: n.noteId })),
+        tasks: tasks.map((t) => ({ id: t.taskId })),
+        emails: [], // I don't think we can unmerge these after moving to them attributes
```

Comment:
I think we can remove this field from `IMemberUnmergePreviewResult`

Reply:
Ok